hw/ip/aes/model/aes_model.core

```diff
@@ -0,0 +1,26 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:model:aes:0.5"
+description: "AES Model"
+filesets:
+  files_cc:
+    files:
+      - crypto.c
+      - crypto.h: { is_include_file: true }
+      - aes.c
+      - aes.h: { is_include_file: true }
+    file_type: cppSource
+
+targets:
+  default:
+    filesets:
+      - files_cc
```

Comment:
should this be `files_dv` for the sake of VCS

Reply:
Changed to `files_dv`